Makefile

```diff
@@ -89,9 +91,9 @@ test_if_changed: cover.out
 # Run tests
 cover.out: ${GO_ALL} ${MANIFESTS}
 
-test:
+test: $(GINKGO)
 ifneq "$(SKIP_TEST)" "1"
-	go test ${go_test_flags} ./... -coverprofile cover.out --ginkgo.parallel.total $(GINKGO_PARALLEL)
+	$(GINKGO) -p --slow-spec-threshold=10s -keep-going ${go_test_flags} ./... -coverprofile cover.out
```

Comment:
Using `go test` has the drawback that we can't use the parallelise feature of ginkgo but it might be worth to not force everyone to install ginkgo.